installer/templates/new/priv/gettext/en/LC_MESSAGES/errors.po

```diff
@@ -0,0 +1,101 @@
+msgid ""
+msgstr ""
+"Language: en\n"
+
+# # Gettext Usage
+#
+# ## .pot files
+#
+#  .pot files hold your translation msgid's for Gettext
+#
+#  .pot entries are *extracted from source automatically*.
+#  You should only add msgid entries to .pot files for dynamic
+#  keys that are not compile-time literals, such as Ecto msgid's.
+#  The msgstr's for .pot entries will always be blank strings.
+#
+# ## .po files
+#
+# Translations for the msgid's defined in .pot files go in the
+# corresponding .po file, located in:
+#
+#     /priv/gettext/[locale]/LC_MESSAGES/[domain].po
+#
+#  where `locale` could be "en" and `domain` "default", "errors"
+#
+#  For .po files, you only enter the msgstr translations.
+#  The msgid's are extracted and merged into the po files for you.
+#
+# See http://hexdocs.pm/gettext for detailed docs
+
+# Ecto validations
+msgid "is invalid"
+msgstr ""
+
+msgid "can't be blank"
+msgstr ""
+
+msgid "has invalid format"
+msgstr ""
+
+msgid "has an invalid entry"
+msgstr ""
+
+msgid "is reserved"
+msgstr ""
+
+msgid "does not match confirmation"
+msgstr ""
+
+## constraints
+msgid "is still associated to this entry"
+msgstr ""
+
+msgid "are still associated to this entry"
+msgstr ""
+
+## length
+msgid "should be %{count} character(s)"
+msgid_plural "should be %{count} character(s)"
+msgstr[0] ""
+msgstr[1] ""
+
+msgid "should have %{count} item(s)"
+msgid_plural "should have %{count} item(s)"
+msgstr[0] ""
+msgstr[1] ""
+
+msgid "should be at least %{count} character(s)"
+msgid_plural "should be at least %{count} character(s)"
+msgstr[0] ""
+msgstr[1] ""
+
+msgid "should have at least %{count} item(s)"
+msgid_plural "should have at least %{count} item(s)"
+msgstr[0] ""
+msgstr[1] ""
+
+msgid "should be at most %{count} character(s)"
+msgid_plural "should be at most %{count} character(s)"
+msgstr[0] ""
+msgstr[1] ""
+
+msgid "should have at most %{count} item(s)"
+msgid_plural "should have at most %{count} item(s)"
+msgstr[0] ""
+msgstr[1] ""
+
+## numbers
+msgid "must be less than %{count}"
```

Comment:
Because those use "count: 1", we are also treating them like plural in our helper. So we need "msgid_plural"